packages/lib-utils/src/app/AppInitSDK.tsx

```diff
@@ -18,6 +20,12 @@ export type AppInitSDKProps = {
   };
 };
 
+export const createActions = (dispatch: Dispatch): ApiActions => ({
```

Comment:
@karelhala
1. I'm wondering if we should rename this function to something specific to API discovery. I'm not sure we would want to encourage exposing all of our actions. It appears all of the actions below are only utilized by API discovery:
  * https://github.com/openshift/dynamic-plugin-sdk/search?q=setResourcesInFlight
  * https://github.com/openshift/dynamic-plugin-sdk/search?q=setBatchesInFlight
  * https://github.com/openshift/dynamic-plugin-sdk/search?q=receivedResources
2. Should we move this export into the API discovery implementation versus `AppInitSDK`?